.github/workflows/ci.yml

```
@@ -202,6 +202,10 @@ jobs:
       run: .github/scripts/check_docs.sh
     - name: Test gifs
       run: gifs/generate_gifs.sh $(ls gifs/scenarios/)
+    - name: Build binary
+      run: ./mill copyTo cli.nativeImage ./scala-cli 
```

Better use the JVM launcher, it should be much faster to generate:
```suggestion
      run: ./mill copyTo cli.launcher ./scala-cli 
```